tenant/store.go

```diff
@@ -38,10 +40,11 @@ func (tenantStore *tenantStore) Entities() []interface{} {
 }
 
 type Tenant struct {
-	Id       uint64 `sql:"AUTO_INCREMENT"`
-	Name     string
-	Segments []Segment
-	Seq      uint64
+	ID         uint64 `sql:"AUTO_INCREMENT"`
+	ExternalID string `sql:"not null;unique"`
```

Comment:
This is fine, it's, I think, is a matter of picking one way and sticking to it - understanding that using a foreign ID may or may not match multiple items?